circup/command_utils.py

```diff
@@ -8,6 +8,7 @@
 import ctypes
 import glob
 import os
+import pickle
```

Comment:
I would be in favor using something other than pickle. It can be unsafe depending on the data that is loaded with it. The python docs contain a warning against using it.
![image](https://github.com/user-attachments/assets/ca6a7c2b-7f3e-49df-99f8-161917f4c75d)

It looks to me like `bundle_examples` dictionary and everything it contains could be serialized as `json` instead. If that is possible I'd be in favor of using it instead of pickle.